test/test_library.py

```diff
@@ -708,13 +708,13 @@ def test_if_def_false_complete(self):
         self._assert_dest(b"/base/not_played")
 
     def test_first(self):
-        self.i.genres = "Pop; Rock; Classical Crossover"
-        self._setf("%first{$genres}")
+        self.i.sub_genres = "Pop; Rock; Classical Crossover"
+        self._setf("%first{$sub_genres}")
         self._assert_dest(b"/base/Pop")
 
     def test_first_skip(self):
-        self.i.genres = "Pop; Rock; Classical Crossover"
-        self._setf("%first{$genres,1,2}")
+        self.i.sub_genres = "Pop; Rock; Classical Crossover"
+        self._setf("%first{$sub_genres,1,2}")
```

Comment:
The rest of the PR references `genres`, this file is the only mention of `sub_genres`. Was this an oversight?

Reply:
The tests are testing the `%first{}` function extracting from a string split on `; `. Genres was the tag used for the tests initially, but now genres is a `MULTI_VALUE_DSV` which is separated by null characters instead. 

Rather than rewriting the tests both to the full `%first{text,count,skip,sep,join}` format to specify the null character separator, which would be changing whats being tested, the original author just swapped out the genres tag for a nonexistent sub_genres tag which would continue to work as a standard string for `%first` to act on.

Reply:
Could you rename this field to something like `semicolon_delimited_field` to remove any confusion and relation to the `genres` field?